modules/contrib/site_verify/site_verify.admin.inc

```diff
@@ -180,11 +185,49 @@ function site_verify_edit_form($form, &$form_state, $record = array(), $engine =
 }
 
 /**
- * Validation callback; save the uploaded file and check file name uniqueness.
+ * Validation callback; sanitize metatag and save the uploaded file
+ */
+function site_verify_validate($form, &$form_state) {
+  if (isset($form_state['values']['meta']) && $form_state['values']['meta']) {
+    site_verify_sanitize_metatag($form, &$form_state);
+  }
+
+  if (isset($form_state['values']['file']) && $form_state['values']['file']) {
+    site_verify_validate_file($form, &$form_state);
+  } else {
+    $form_state['values']['file_contents'] = NULL;
+  }
+
+}
+
+/**
+ * Sanitize metatag input
+ */
+function site_verify_sanitize_metatag($form, &$form_state) {
+  $valid_metatag_set = FALSE;
+
+  $html = $form_state['values']['meta'];
+  $dom = new DOMDocument;
+  $dom->loadHTML($html);
+  //@TODO: If we only care about the first metatag... why foreach?  why not [0]?
+  foreach ($dom->getElementsByTagName('meta') as $tag) {
+    if ($tag->getAttribute('name') && $tag->getAttribute('content')) {
```

Comment:
If you add `<script type="text/javascript">alert('test');</script>` to the begining or end of a metatag, it is stripped and replaced with just the valid metatag.  If you only add` <script type="text/javascript">alert('test');</script>`, you get an error that a valid metatag isn't found.